x-pack/platform/plugins/shared/observability_ai_assistant/server/service/client/operators/continue_conversation.ts

```diff
@@ -186,8 +186,8 @@ export function continueConversation({
   chat: AutoAbortedChatFunction;
   signal: AbortSignal;
   functionCallsLeft: number;
-  adHocInstructions: AdHocInstruction[];
-  userInstructions: Instruction[];
+  apiUserInstructions: InstructionOrPlainText[];
```

Comment:
Isn't this handled at the API layer?
```suggestion
  apiUserInstructions: Instruction[];
```